ChangeLog

```diff
@@ -1,52 +1,65 @@
-07/09/2016 - compatibility with Coq 8.7 and several small fixes -
-	version 1.6.2 and upcomming version 1.7
-	* Compatibility with Coq 8.7
+23/04/2018 - compatibility with Coq 8.7 and several small fixes -
+	upcoming version 1.7
+	* Compatibility with Coq 8.8 (and >= 8.6)
 	* Lost compatibility with Coq 8.4
 
-07/09/2016 - compatibility with Coq 8.7 and several small fixes -
-	upcomming version 1.7
-	* New Theorems:
-	   dec_factor_theorem, abstract_context,
-	   mul_bin_down, mul_bin_left
+	* Integration to Coq: ssrbool.v ssrfun.v and plugin.
+	  ssrtest also moved to Coq test suite.
+
+	* Library refactoring:
+	odd_order and real_closed: are now in separate repositories.
```

Comment:
I think the style is to indent as in 
```
   * fooo
     bar
```

Reply:
Yeah, I've been lazy for this draft and kept what Emacs did for me... I will correct the style.